packages/file-asset-apis/test/s3/helpers.ts

```diff
@@ -17,11 +17,9 @@ export { MINIO_HOST, MINIO_ACCESS_KEY, MINIO_SECRET_KEY };
 export async function makeClient() {
     return createS3Client({
         endpoint: MINIO_HOST,
-        credentials: {
-            accessKeyId: MINIO_ACCESS_KEY,
-            secretAccessKey: MINIO_SECRET_KEY,
-        },
-        maxAttempts: 4,
+        accessKeyId: MINIO_ACCESS_KEY,
```

Comment:
Does this need to be rolled back to `credentials`?  Or, I guess this is a test helper and just testing one possible way to supply credentials.

Actually, I'm confused on this point, what way or ways do we supply credentials to `createS3Client`?

Reply:
Given this type:

```
export interface S3ClientConfig extends BaseConfig {
    sslEnabled?: boolean,
    certLocation?: string,
    secretAccessKey?: string,
    accessKeyId?: string,
    maxRetries?: number
}
```

Can't we supply credentials using either method?

```
{
    secretAccessKey: "foo",
    accessKeyId: "bar",
}
```

or

{
    credentials: {
        secretAccessKey: "foo",
        accessKeyId: "bar"
    }
}

Reply:
Prior to this PR `createS3Client` was able to take credentials either way.  We can't use the credentials object when describing the connector because it's not valid according to the terafoundation schema.

Reply:
I reverted that test and added a test with the credentials object style.